streaming/java/streaming-state/src/main/java/org/ray/streaming/state/keystate/desc/MapStateDescriptor.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ray.streaming.state.keystate.desc;
+
+import java.util.Map;
+import org.ray.streaming.state.keystate.state.MapState;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * MapStateDescriptor.
+ */
+public class MapStateDescriptor<K, V> extends AbstractStateDescriptor<MapState<K, V>, Map<K, V>> {
+
+  private static Logger LOG = LoggerFactory.getLogger(MapStateDescriptor.class);
+
+  public MapStateDescriptor(String name, Class<K> keyType, Class<V> valueType) {
+    super(name, null);
+    LOG.debug("keyType {}, valueType {}", keyType, valueType);
```

Comment:
`keyType` & `valueType` isn't used expect for logging. Why do we have to log here? It seems like a temporary debugger but polluted into the final commits.

Reply:
And it is weird we use `null` when calling `super`. It looks should be a meaning type information.

Reply:
ok, I'll change the codes